packages/components/src/styles/footnote/footnote.stories.ts

```diff
@@ -0,0 +1,147 @@
+      url: 'https://www.figma.com/file/DeZ6iJggYfpSXHcEexNeYs/Footnote?type=design&node-id=0-1&mode=design&t=7vjD8Qq8iKSnKn6a-0'
+    }
+  },
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'default',
+      value:
+        'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.'
+    }
+  ]),
+  argTypes,
+  decorators: [
+    (story: () => typeof html) => html`
+      <style>
+        td.template {
+          text-align: left;
+        }
+      </style>
+      ${story()}
+    `
+  ]
+};
+
+/**
+ * Default: This shows sd-footnote in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({
+      options: { templateContent: '<div class="sd-container"><p class="%CLASSES%">%SLOT%</p></div?' },
+      args
+    });
+  }
+};
+
+/**
+ * Use the `--number` class to display the footnote as a numbered list.
+ */
+
+export const Number = {
+  parameters: { controls: { exclude: ['sd-footnote--number'] } },
+
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: [{ type: 'attribute', name: 'sd-footnote--number', values: [false, true] }]
+      },
+      options: { templateContent: '<div class="sd-container"><p class="%CLASSES%">%SLOT%</p></div?' },
+      args
+    });
+  }
+};
+
+/**
+ * Use the `--reset` class to reset the number of the footnote and start counting from 1. This should be used at the beginning of a counting sequence and can be used again when starting a new context.
+ */
+
+export const Reset = {
+  parameters: { controls: { include: [] } },
+
+  render: () => {
+    return html`<div class="sd-container flex flex-col gap-4">
+    <p class="sd-footnote sd-footnote--number sd-footnote--reset">
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.
+    </p>
+
+    <p class="sd-footnote sd-footnote--number">
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.
+    </p>
+
+    <p class="sd-footnote sd-footnote--number">
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.
+    </p>
+
+    <h4 class="sd-headline sd-headline--size-base">New Context</h4>
+
+    <p class="sd-footnote sd-footnote--number sd-footnote--reset">
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.
+    </p>
+
+    <p class="sd-footnote sd-footnote--number">
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliq erat, sed diam voluptua.
+    </p>
+    </div>
+    </div>`;
+  }
+};
+
```

Comment:
I  find it a bit confusingI that when resetting, with the content always being the same, it's not easy to discern what exactly has been altered. Could you include various Lorem Ipsum texts so that it's clearer how the reset functions and what changes occur?